components/Toast.js

```diff
@@ -1,12 +1,21 @@
 const toast = (message, type) => {
   const toastWrapper = document.createElement("div");
-  toastWrapper.classList.add("toast-wrapper");
+  toastWrapper.className = "toast-wrapper";
+
+  setTimeout(() => {
+    toastWrapper.className += " animation";
+  }, 1);
+
   toastWrapper.innerText = message;
   toastWrapper.style.background = "#0ec277";
   if (type === "error") {
     toastWrapper.style.background = "#db072a";
   }
 
+  setTimeout(() => {
+    toastWrapper.classList.remove("animation");
+  }, 1800);
```

Comment:
`Magic number`. Can we please replace it with a const?

i.e: A magic number in programming often refers to use of arbitrary decimal values in code. To make the code more understandable and enjoyable for other contributor, it's recommended to avoid magic numbers and use const.
An example is given below:

```js
someFunction(1)

// What is 1 in the above function call. It's very vague right now.
```

A better way is:

```js
import { DELAY_SECOND } from "../const.js"

const delayTime = 1 * DELAY_SECOND
someFunction(delayTime)

// Now it's more clear
```

```suggestion
  }, TOAST_MESSAGE_ANIMATION_DELAY);
```

Reply:
Can't agree with you. The second argument of `setTimeout()` function is always time delay in milliseconds. It's not a magic number as it's already know value. The example you've given depicts a real magic number. No clue of what `1` means here but not in my case. What do you say?

Reply:
Now I see that example I've given, It also seems to me that, this might not be a case of magic number. 
Addressing that first, my original suggestion still stands for other good practices such as, this delay number could have been used in multiple cases and if we would decide to change it to something else, like `1700` we'd have to make the changes in `n` number of places. Making it a const will centralise it.